From: "2329574154@qq.com" <2329574154@qq.com>
Date: Sat, 3 May 2025 05:56:24 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 100.0% (29 of 29 strings)

Translation: openkylin-nile-new/kylin-os-manager-session-service
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-session-service/vi/
---
 .../kylin-os-manager-session-service_vi.ts         | 28 +++++++++++-----------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/daemons/sessionservice/translations/kylin-os-manager-session-service_vi.ts b/daemons/sessionservice/translations/kylin-os-manager-session-service_vi.ts
index 477d8c2..3b99737 100644
--- a/daemons/sessionservice/translations/kylin-os-manager-session-service_vi.ts
+++ b/daemons/sessionservice/translations/kylin-os-manager-session-service_vi.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../disk_monitor_exception_popup.cpp" line="25"/>
         <source>Kylin Os Manager</source>
-        <translation>Quản gia Kylin</translation>
+        <translation>Trình quản lý Kylin&#xa0;OS</translation>
     </message>
     <message>
         <location filename="../disk_monitor_exception_popup.cpp" line="54"/>
@@ -29,7 +29,7 @@
     <message>
         <location filename="../driverinstall/aptinstaller.cpp" line="68"/>
         <source>Install timeout.</source>
-        <translation>Cài đặt hết thời gian chờ</translation>
+        <translation>Quá thời gian cài đặt.</translation>
     </message>
 </context>
 <context>
@@ -37,12 +37,12 @@
     <message>
         <location filename="../service_support_popup.cpp" line="22"/>
         <source>Kylin Os Manager</source>
-        <translation>Quản lý Kylin</translation>
+        <translation>Trình quản lý Kylin OS</translation>
     </message>
     <message>
         <location filename="../service_support_popup.cpp" line="40"/>
         <source>Detected system abnormality, please contact the technical service immediately or call 400!</source>
-        <translation>Phát hiện hệ thống bất thường, vui lòng liên hệ nhân viên kỹ thuật hoặc gọi số 400 ngay lập tức!</translation>
+        <translation>Phát hiện sự cố bất thường trong hệ thống, xin vui lòng liên hệ ngay với dịch vụ kỹ thuật hoặc gọi số 400 ngay lập tức!</translation>
     </message>
     <message>
         <location filename="../service_support_popup.cpp" line="57"/>
@@ -57,12 +57,12 @@
     <message>
         <location filename="../service_support_popup.cpp" line="67"/>
         <source>Mail</source>
-        <translation>Mail</translation>
+        <translation>Email</translation>
     </message>
     <message>
         <location filename="../service_support_popup.cpp" line="89"/>
         <source>For common questions, please refer to</source>
-        <translation>Vấn đề thường gặp vui lòng thăm khảo</translation>
+        <translation>Vấn đề thường gặp, vui lòng thăm khảo</translation>
     </message>
     <message>
         <location filename="../service_support_popup.cpp" line="92"/>
@@ -92,7 +92,7 @@
         <location filename="../sessionclient.cpp" line="444"/>
         <location filename="../sessionclient.cpp" line="447"/>
         <source>Kylin Os Manager</source>
-        <translation>Quản gia Kylin</translation>
+        <translation>Trình quản lý Kylin OS</translation>
     </message>
     <message>
         <location filename="../sessionclient.cpp" line="121"/>
@@ -105,7 +105,7 @@
     <message>
         <location filename="../sessionclient.cpp" line="122"/>
         <source>The system detects that a new device has an installable driver. Do you want to install?</source>
-        <translation>Đã phát hiện thiết bị mới có thể cài đặt trình điều khiển. Bạn có muốn cài đặt không?</translation>
+        <translation>Đã phát hiện thiết bị trình điều khiển mới có thể cài đặt, bạn có muốn cài đặt trình điều khiển không?</translation>
     </message>
     <message>
         <location filename="../sessionclient.cpp" line="123"/>
@@ -130,17 +130,17 @@
     <message>
         <location filename="../sessionclient.cpp" line="219"/>
         <source>Open Scanner</source>
-        <translation>Kích hoạt quét</translation>
+        <translation>Mở chức năng quét</translation>
     </message>
     <message>
         <location filename="../sessionclient.cpp" line="271"/>
         <source>Driver install failed.</source>
-        <translation>Cài đặt trình điều khiển không thành công.</translation>
+        <translation>Cài đặt trình điều khiển thất bại.</translation>
     </message>
     <message>
         <location filename="../sessionclient.cpp" line="349"/>
         <source>Abnormal writing of terminal log, please contact the administrator</source>
-        <translation>Phát hiện lỗi ghi nhật ký thiết bị, vui lòng liên hệ quản trị viên</translation>
+        <translation>Ghi nhật ký đầu cuối bất thường, vui lòng liên hệ với quản trị viên</translation>
     </message>
     <message>
         <location filename="../sessionclient.cpp" line="352"/>
@@ -157,17 +157,17 @@
     <message>
         <location filename="../sessionclient.cpp" line="386"/>
         <source>There is an abnormal writing of logs in the system. Please go to Kylin Os Manager for cleaning!</source>
-        <translation>Phát hiện lỗi ghi nhật ký bất thường trong hệ thống , vui lòng truy cập quản gia Kylin để dọn dẹp!</translation>
+        <translation>Hệ thống ghi nhật ký bất thường, vui lòng truy cập trình quản lý Kylin OS để dọn dẹp!</translation>
     </message>
     <message>
         <location filename="../sessionclient.cpp" line="485"/>
         <source>The System disk space is tight, please go to Kylin Os Manager for cleaning!</source>
-        <translation>Dung lượng ổ cứng hệ thống sắp đầy, vui lòng truy cập quản gia Kylin để dọn dẹp!</translation>
+        <translation>Không gian ổ đĩa hệ thống sắp hết, vui lòng truy cập trình quản lý Kylin OS để dọn dẹp!</translation>
     </message>
     <message>
         <location filename="../sessionclient.cpp" line="448"/>
         <source>The avaliable disk space is less than 20%, please clean it up in a timely manner</source>
-        <translation>Dung lượng ổ cứng khả dụng dưới 20%,vui lòng dọn dẹp kịp thời</translation>
+        <translation>Không gian ổ đĩa khả dụng dưới 20%, vui lòng dọn dẹp kịp thời</translation>
     </message>
 </context>
 </TS>
